mythril/laser/ethereum/state.py

```diff
@@ -90,19 +90,23 @@ def get_word_at(self, index: int):
 
     def __getitem__(self, item):
         if isinstance(item, slice):
+            start, step, stop = item.start, item.step, item.stop
```

Comment:
What required the change here?

Reply:
if step in slice is unintialized then it is None, in many cases it was uninitialized, which led to Z3Exception